include/common/message_queues.h

```diff
@@ -110,4 +110,11 @@ Message queues used across MME, S1ap, S11, S6a
 
 /********* STAGE 3 DETACH END    ***************/
 
+/********* PAGING MESSAGE ***************/
+
+#define S6A_DDN_QUEUE  "/tmp/s1ap/s6ap_DDN_Q"
```

Comment:
Queue name should be S11_DDN_QUEUE. S6A is misleading.

Reply:
Yes. Correcting it.
Changing the directory path also to s11